jsk_arc2017_baxter/launch/setup/baxter.launch

```diff
@@ -8,6 +8,16 @@
   <arg name="launch_right" default="true" />
   <arg name="respawn_arduino" default="true" />
 
+  <arg name="SHELF_CONFIG_FILE"
+       default="$(find jsk_arc2017_baxter)/config/$(arg task)/shelf_marker.yaml"
```

Comment:
`task` argument is evaluated here, but `baxter.launch` doesn't have that argument and causes an error.